t/lib/TestExtension.pm

```diff
@@ -0,0 +1,13 @@
+package TestExtension;
```

Comment:
Could you please expand this TextExtension module so that it actually writes a file into the docroot, based on the source file that it reads? That will allow the tests to pass without needing to modify the "+4" magic number (which is has changed to "+5" in more recent master-branch work, anyway) and would also serve to demonstrate a little more clearly how an extension might actually work in practice.